test/fibonacci.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai'
+import fibonacci from '../src/fibonacci'
+
+describe.only('fibonacci()', function(){
+
+  it('should be a function', function(){
+    expect(fibonacci).to.be.a('function')
+  })
+
+  it('should return an array of length 50, whos 9th value is 34', function(){
+    const fib = fibonacci(50)
+    expect(fib).to.be.an('array')
+    expect(fib.length).to.equal(50)
+    expect(fib[9]).to.equal(34)
```

Comment:
also if giving `fibonacci` and integer should match the length of the returned array than test that explicitly. 

```js

context("when given 20", function(){
  it("should return an array of length 20", function(){
    expect(fibonacci(20)).to.have.length(20)
  })
})


context("when given 30", function(){
  it("should return an array of length 30", function(){
    expect(fibonacci(30)).to.have.length(30)
  })
})
```